scripts/martech.js

```diff
@@ -0,0 +1,41 @@
+import { getMetadata, loadScript } from './aem.js';
+// eslint-disable-next-line import/no-cycle
+import { getLocaleAsBCP47 } from './scripts.js';
+
+function initDataLayer() {
+  window.digitalData = {
+    page: {
+      pageInfo: {
+        brand: 'sid',
+        pageName: getMetadata('og:title'),
+        destinationURL: window.location.href,
+        referringURL: document.referrer,
+        language: getLocaleAsBCP47(),
+        pageURL: window.location.href,
+        pageType: 'homepage', // TODO: parse pagetype dynamically?
```

Comment:
The previous implementation used the template name for `pageType`. For example [this location page](https://www.shredit.com/en-us/service-locations/omaha) on the prod site has a meta tag `<meta name="template" content="location"/>`

Reply:
Yes, indeed. But it won't be a 1:1 match. I don't know yet what the pageType property is used for in you rules. Depending on that the results may vary. But I'm happy to use the template metadata as a replacement for now; just expecting for things to break.